fe/fe-core/src/main/java/com/starrocks/connector/delta/CachingDeltaLakeMetastore.java

```diff
@@ -137,10 +140,6 @@ public MetastoreTable getMetastoreTable(String dbName, String tableName) {
 
     @Override
     public Table getTable(String dbName, String tableName) {
-        if (ConnectContext.get() != null && ConnectContext.get().getCommand() == MysqlCommand.COM_QUERY) {
```

Comment:
why have to move to `getCachedSnapshot`?  Should iceberg table also handle updating last access time in the same way?